### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
## What does this PR do?
- Adds specific test to ensure it queries entities based on a specific interval.

Resolves [#EM-209](https://rootstrap.atlassian.net/browse/EM-209?atlOrigin=eyJpIjoiNmFhNjZjY2UxZmQ5NDdjMjg1ZjdmZDNiYmVlNWI4NjciLCJwIjoiaiJ9)

### CODE REVIEW

File: spec/services/metrics/review_turnaround/per_project_spec.rb [hunk 1]
```diff
@@ -18,6 +18,18 @@
       end
 
       it_behaves_like 'available metrics data'
+
+      context 'when interval is set' do
+        let(:subject) { described_class.call(project.id, interval) }
+
+        before do
+          rr = create(:review_request, project: project)
```
Comment (martinjaimem) [inline spec/services/metrics/review_turnaround/per_project_spec.rb:26]:
```suggestion
          review_request = create(:review_request, project: project)
```

File: spec/services/metrics/merge_time/per_language_spec.rb [hunk 2]
```diff
@@ -24,6 +24,19 @@
       end
 
       it_behaves_like 'available metrics data'
+
+      context 'when interval is set' do
+        let(:subject) { described_class.call([ruby_lang.id, python_lang.id], interval) }
+
+        before do
+          projects.each do |project|
+            pr = create(:pull_request, project: project, merged_at: 5.weeks.ago)
```
Comment (martinjaimem) [inline spec/services/metrics/merge_time/per_language_spec.rb:33]:
```suggestion
            pull_request = create(:pull_request, project: project, merged_at: 5.weeks.ago)
```

**Review by martinjaimem — APPROVED** [review]
👍 🚀

File: spec/support/shared_examples/metrics.rb [hunk 3]
```diff
@@ -28,6 +28,20 @@
   end
 end
 
+RSpec.shared_examples 'metric value unchanged' do
+  let(:current_time) { Time.zone.now }
+  let(:interval) do
+    (current_time - 4.weeks).beginning_of_week..current_time.end_of_week
```
Comment (santib) [inline spec/support/shared_examples/metrics.rb:34]: `(current_time - 4.weeks)` => `4.weeks.ago`

**Review by santib — APPROVED** [review]

**Pull request merged**